rest-api-spec/src/main/resources/rest-api-spec/test/bulk/20_list_of_strings.yml

```diff
@@ -11,6 +11,8 @@
 
   - do:
       count:
+        # we count through the primary in case there is a replica that has not yet fully recovered
```

Comment:
Removing this since mixed cluster tests are failing as the changes are not BWC/MixedCluster compatible yet.